### PR TITLE
Add more Pact context before code example

### PR DESCRIPTION
**Pull request by bts**

### CODE REVIEW

File: src/pages/introducing-the-pact-property-checker/index.md [hunk 1]
```diff
@@ -1,20 +1,54 @@
 ---
-title: Introducing the Pact analysis tools
+title: Introducing the Pact property checker
 date: 2018-05-17 13:16:01
 tags:
 ---
 
-Together with [Kadena](http://kadena.io/), [Monic](https://www.monic.co/) has developed new a static analysis tool for the [Pact](https://github.com/kadena-io/pact) smart contract language. In this post we'll talk about the purpose of our tool, what it can do today, and what we have planned for the future.
+Together with [Kadena](http://kadena.io/), [Monic](https://www.monic.co/) has developed new a static analysis tool for the [Pact](https://github.com/kadena-io/pact) smart contract language that we're calling the "property checker." In this post we'll talk about the purpose of our tool, what it can do today, and what we have planned for the future.
```
Comment (joelburget) [inline src/pages/introducing-the-pact-property-checker/index.md:7]: Not a big deal, but if we're setting a precedent here I prefer "logical quotes".

File: src/pages/introducing-the-pact-property-checker/index.md [hunk 2]
```diff
@@ -66,19 +100,19 @@ type M = StateT (SFunArray String String) (WriterT [SString] Symbolic)
...
+                         tell [q']
+                         lift $ lift exists_
+eval (Const str)    = return $ literal str
+eval (Concat e1 e2) = (.++) <$> eval e1 <*> eval e2
+eval (ReadVar nm)   = do n   <- eval nm
+                         arr <- get
+                         return $ readArray arr n
 ```
 
-`SFunArray` represents a mapping (think of a block of memory or a database table). Our `SFunArray String String` represents the variables in scope in our language. We also write our program's queries as `[SString]` (`SString` is an SBV *symbolic* string).
```
Comment (joelburget) [inline src/pages/introducing-the-pact-property-checker/index.md:79]: (same re logical parens).

**Review by joelburget — APPROVED** [review]

File: src/pages/introducing-the-pact-property-checker/index.md (hunk 1 above)
Comment (bts) [reply src/pages/introducing-the-pact-property-checker/index.md:7]: Cool -- didn't know there was a term for it! Apparently wikipedia uses logical style. This sounds good to me.

Comment (bts) [reply src/pages/introducing-the-pact-property-checker/index.md:7]: I was always taught that _The Elements of Style_ is the only way, but it's painful to transition to that from the programming world.

**Pull request merged**